administrator/components/com_associations/helpers/associations.php

```diff
@@ -83,6 +83,16 @@ public static function getComponentProperties($component = '')
 			$properties->fields->access      = isset($properties->tableFields['access']) ? 'access' : null;
 			$properties->fields->published   = isset($properties->tableFields['state']) ? 'state' : null;
 			$properties->fields->published   = isset($properties->tableFields['published']) ? 'published' : $properties->fields->published;
+			
+			if (isset($properties->tableFields['created_by']))
+			{
+				$properties->fields->created_by  = 'created_by';
+			}
+			else
+			{
+				$properties->fields->created_by = isset($properties->tableFields['created_user_id']) ? 'created_user_id' : null;
+			}
+			
```

Comment:
forget it. i already added this in my merged PR, please remove this changres so you don't get conflicts